src/components/ConfirmModal/index.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import {
+    TouchableOpacity, Text, View,
+} from 'react-native';
+import PropTypes from 'prop-types';
+import Header from '../Header';
+import Modal from '../Modal';
+import styles from '../../styles/styles';
+import CONST from '../../CONST';
+import withWindowDimensions from '../withWindowDimensions';
+
+const propTypes = {
+    /** Title of the modal */
+    title: PropTypes.string.isRequired,
+
+    /** A callback to call when the form has been submitted */
+    onConfirm: PropTypes.func.isRequired,
+
+    /** A callback to call when the form has been closed */
+    onCancel: PropTypes.func.isRequired,
+
+    /** If the screen is small */
+    isSmallScreenWidth: PropTypes.bool.isRequired,
```

Comment:
In most other places in the codebase, we typically just do `...windowDimensionsPropTypes`

Reply:
Sure. will do that.